### PR TITLE
Add doc: how to activate plugin from cmdline

### PR DESCRIPTION
**Pull request by delassio**
Alternative way to activate plugin
for more info check Tuleap-devel discussion :
https://tuleap.net/pipermail/tuleap-devel/2019-January/005951.html

### CODE REVIEW

File: languages/en/installation-guide/install-plugins.rst [hunk 1]
```diff
@@ -32,6 +32,13 @@ is the name of the package of the plugin you want to install):
 
 Once the plugin is installed, go to Tuleap site admin home page and enter the plugin administration page. You can now
 install and activate the new plugin from 'Not yet installed' tab.
+it's also possible to install and activate new plugin from command line You can use:
```
Comment (vaceletm) [inline languages/en/installation-guide/install-plugins.rst]: It's not very useful to have the command listed twice. You can replace this sentence by "You can also activate the plugin with command line with command hereafter: "

Comment (delassio) [reply languages/en/installation-guide/install-plugins.rst]: my sentence: You can also activate the plugin from command line with:

File: languages/en/installation-guide/install-plugins.rst [hunk 2]
```diff
@@ -32,6 +32,13 @@ is the name of the package of the plugin you want to install):
 
 Once the plugin is installed, go to Tuleap site admin home page and enter the plugin administration page. You can now
 install and activate the new plugin from 'Not yet installed' tab.
+You can also activate the plugin from command line with: 
+"/usr/share/tuleap/src/utils/php-launcher.sh /usr/share/tuleap/tools/utils/admin/activate_plugin.php awesomestuff" as codendiadm
```
Comment (vaceletm) [inline languages/en/installation-guide/install-plugins.rst]: You don't need this line, you already have it below

Comment (vaceletm) [reply languages/en/installation-guide/install-plugins.rst]: Just remove the one line L36, as you already have the command well formatted L40-41

Comment (delassio) [reply languages/en/installation-guide/install-plugins.rst]: done, thanks for advice, for centos 6 i havn't tested yet 
as the support will be deprecated should we remove centos 6 (install rpm)
or add the command for (activate plugin)

Comment (vaceletm) [reply languages/en/installation-guide/install-plugins.rst]: It's the same command and works the same way so no worries

**Pull request merged**